webapi/src/main/scala/org/knora/webapi/responders/v1/ValuesResponderV1.scala

```diff
@@ -1399,11 +1466,20 @@ class ValuesResponderV1 extends ResponderV1 {
 
             response <- (storeManager ? SparqlSelectRequest(sparqlQuery)).mapTo[SparqlSelectResponse]
             rows: Seq[VariableResultsRow] = response.results.bindings
-        } yield sparqlQueryResults2LinkValueQueryResult(rows, userProfile)
+            maybeLinkValueQueryResult = sparqlQueryResults2LinkValueQueryResult(rows, userProfile)
```

Comment:
The resources responder already calls the special `PermissionUtilV1` method to check permissions on link values when necessary, so permissions on standoff links are handled correctly there. So does the search responder. I had just forgotten to use that method in one place in the values responder (because I had forgotten that you could get a link value via the normal route for getting values, not just through the special route for getting link values). This shows that having a special method just to check permissions on link values is a bad design, because it's easy to forget to use it. But I'd rather get the release done than refactor this now. Let's design it better for API v2.